### DISCUSSION TITLE
GSOC 2024: Excited to Contribute: Image-to-Image and Image-to-Text Generation GenAI Project

### DISCUSSION THREAD

**Discussion by AishwaryaDekhane** [openvinotoolkit/openvino#22713] [Google Summer of Code]
I'm really interested in this project! The addition of Image-to-Image and Image-to-Text generation with GUI support for OpenVINO GenAI sounds like a fantastic opportunity to delve into some cutting-edge technology. I have experience with C++ programming and a good understanding of PyTorch, Qt Creator. Looking forward to potentially being a part of it!

**Comment by adrianboguszewski** [comment]
Thanks for being interested. You will need to fill out our template from https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code and submit it in the GSoC web app. But, of course, we need to wait until the official Google announcement.

**Reply by AishwaryaDekhane** [reply]
Thank you so much @adrianboguszewski for getting back to me! I've taken note of the template on the OpenVINO GitHub page for the Google Summer of Code application. I'll make sure to fill it out as soon as the official announcement is made by Google.

In the meantime, could you please guide me on how I can get in touch with potential mentors for this project? I'd love to discuss it further and ensure that I'm aligned with the goals and requirements. Looking forward to your guidance.

**Reply by adrianboguszewski** [reply]
You should use discussions for this. The mentors are @yangsu2022 and @FionaZZ92

**Comment by ArshCypherZ** [comment]
Hey, perhaps you can share your expertise with a young [text to image](https://enhanceai.art/text-to-image) feature? we really need advises.

**Comment by yangsu2022** [comment]
Thanks for being interested in contributing to OpenVINO C++ GenAI samples for AIPC project.

Besides the related python [OpenVINO notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/b139f75ef4824064fced7c7784a447f0751c774b/notebooks/235-controlnet-stable-diffusion), OpenVINO [CONTRIBUTING](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md) details will be helpful for learning OpenVINO components and the PR to [openvino.genai](https://github.com/openvinotoolkit/openvino.genai).

BTW, the code validation needs on both Windows and Linux platforms.
